app/mailers/user_mailer.rb

```diff
@@ -0,0 +1,10 @@
+class UserMailer < ApplicationMailer
+  default from: 'notifications@bidtastic.com'
+  layout 'mailer'
+
+  def welcome_email(user)
+    @user = user
+    @url  = 'http://bidtastic.com'
```

Comment:
This is weird. Just use the helpers in the view.

`<%= link_to "Bidtastic", root_url %>`

Because this is `hard coding` the URL and you'll have to run through the code fixing all this when the customer changes their domain in the future.. Avoid magic strings and numbers in code as much as much as possible.